templates/component.ejs

```diff
@@ -11,11 +11,10 @@ import View from '<%= props.appName %>/app/components/base/View';
 const styles = StyleSheet.create({
     container: {},
 });
-
-type Props = {};
 <% var typeOfComponent = props.isPure ? 'PureComponent' : 'Component' %>
-<% if (props.isConnected) { %>class <%= props.name %> extends <%= typeOfComponent %><Props> {<% } else { %>export default class <%= props.name %> extends <%= typeOfComponent %><Props> {<% } %>
-
+type Props = {};
+<% var classDeclaration = props.isConnected ? 'class ' + props.name + ' extends ' + typeOfComponent + '<Props> {' : 'export default class ' + props.name + ' extends ' + typeOfComponent + '<Props> {'%>
```

Comment:
🤔 That looks more complicated/less readable than what it should be.

What do you think of : 
```
type Props = {};
<% var classDeclaration = props.isConnected ? 'class' : 'export default class' %>
<%= classDeclaration %> <%= props.name %> extends <%= typeOfComponent %><Props> {
    // Defaults for props
    static defaultProps = {};
```

Reply:
Much more readable. I updated.